package/boot/uboot-mvebu/Makefile

```diff
@@ -63,9 +63,6 @@ UBOOT_TARGETS:= \
 	uDPU
 
 define Build/Configure
-	# enable additional options beyond <device>_defconfig
-	echo CONFIG_CMD_SETEXPR=y >> $(PKG_BUILD_DIR)/configs/$(UBOOT_CONFIG)_defconfig
-
 	$(call Build/Configure/U-Boot)
 endef
```

Comment:
The `include/u-boot.mk` defines `Build/Configure/Default = $$$$(Build/Configure/U-Boot)`. Please remove the `Build/Configure` block here it is not needed any more.

Reply:
Ok, done.